mlog/global_test.go

```diff
@@ -0,0 +1,143 @@
+package mlog_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/mattermost/mattermost-server/mlog"
+)
+
+func TestLoggingBeforeInitialized(t *testing.T) {
+	require.NotPanics(t, func() {
```

Comment:
This test works "in isolation" to verify the behaviour, but I'm not sure if it meaningfully tests anything if another tests globally initializes `mlog`.